packages/seacas/libraries/ioss/src/Ioss_Decomposition.h

```diff
@@ -133,32 +133,44 @@ namespace Ioss {
     bool distributionFactorConstant{false}; // T if all distribution factors the same value.
   };
 
-  IOSS_EXPORT size_t
-  get_all_block_ioss_element_size(const std::vector<BlockDecompositionData> &blocks);
+  class IOSS_EXPORT ElementBlockBatchOffset
+  {
+  public:
+    ElementBlockBatchOffset(const std::vector<BlockDecompositionData> &data)
+    : m_data(data) {}
+
+    ElementBlockBatchOffset()                                = delete;
+    ElementBlockBatchOffset(const ElementBlockBatchOffset &) = delete;
+    ElementBlockBatchOffset(ElementBlockBatchOffset &&)      = default;
 
-  IOSS_EXPORT size_t
-  get_all_block_ioss_offset_size(const std::vector<BlockDecompositionData> &blocks,
-                                 const std::vector<int>                    &block_component_count);
+    ~ElementBlockBatchOffset() = default;
 
-  IOSS_EXPORT std::vector<size_t>
-              get_all_block_ioss_offset(const std::vector<BlockDecompositionData> &blocks,
-                                        const std::vector<int>                    &block_component_count);
+    size_t get_ioss_element_size(const std::vector<int64_t>& blockSubsetIndex) const;
```

Comment:
These probably need `IOSS_EXPORT` so they will work correctly on a windows build.

Reply:
The class itself is already marked with `IOSS_EXPORT` ... do the individual functions in it also need to be marked?